cmd/cli/kubernetes.go

```diff
@@ -108,3 +107,21 @@ func applyManifest(completion string) error {
 	}
 	return nil
 }
+
+func getCurrentContextName() string {
+
+	var kubeConfig string
```

Comment:
can we refactor this to avoid code duplication?
I think we are using similar logic in `applyManifest` https://github.com/sozercan/kubectl-ai/blob/b64b73c01bd2eeb7fa1006d65d25c1e9db4aaa42/cmd/cli/kubernetes.go#L23-L63